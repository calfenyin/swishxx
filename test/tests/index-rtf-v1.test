index | | -d data -e rtf:*.rtf -i rtf.index -r -v1 | . | 0
