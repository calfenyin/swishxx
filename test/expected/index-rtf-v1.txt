
index: done:
  3 indexed
  1459 words, 571 indexed, 199 unique

